packages/dma-library/src/operations/refinance/common/refinance-return-funds.calls.ts

```diff
@@ -0,0 +1,37 @@
+import { actions } from '@dma-library/actions'
+import {
+  WithAaveLikeStrategyAddresses,
+  WithNetwork,
+  WithPositionStatus,
+  WithStorageIndex,
+} from '@dma-library/types/operations'
+import { ActionPathDefinition } from '@dma-library/types/operations-definition'
+
+import { RefinancePartialOperationGenerator } from '../types'
+
+export type RefinanceReturnFundsOperationArgs = WithStorageIndex &
+  WithPositionStatus &
+  WithAaveLikeStrategyAddresses &
+  WithNetwork
+
+export const refinanceReturnFunds_calls: RefinancePartialOperationGenerator = async _args => {
+  const args = _args as RefinanceReturnFundsOperationArgs
+  const { network, position, lastStorageIndex } = args
+
+  const returnDebtFunds = actions.common.returnFunds(network, {
```

Comment:
Should we add `unwrap` here in case of WETH debt?

Reply:
Taken offline: yes, WETH unwrapping has been added